devo/sender/data.py

```diff
@@ -35,16 +35,18 @@ class SenderConfigSSL:
     :param cert:  (str) cert src file
     :param chain:  (str) chain src file
     :param pkcs:  (dict) (path: pfx src file, password: of cert)
+    :param sec_level: (int) default None. If certs are too weak you can change
+    this param to work with it
```

Comment:
What are the valid values here? maybe adding a link to an url can be helpful.

Reply:
Updated in documentation